ci/blackbox/config.json

```diff
@@ -23,7 +23,7 @@
                             "id": "postgres-micro-11",
                             "name": "micro-11",
                             "rds_properties": {
-                                "allocated_storage": 10,
+                                "allocated_storage": 5,
```

Comment:
are the changes to allocated storage intentional?

Reply:
Yes. The integration tests now include a test that deliberately fills the disk space. Lower disk space means a marginal improvement in the integration test run time.

Reply:
Yeah it's to make the plans consistent for the tests.... micro is an aws micro instance with 5g of storage, small is a small instance with 10g of storage.